spyder/app/tour.py

```diff
@@ -94,6 +101,13 @@ def get_tour(index):
     """
     sw = SpyderWidgets
     qtconsole_link = "https://qtconsole.readthedocs.io/en/stable/index.html"
+    #docs_link = __docs_url__
+    docs_link = "https://docs.spyder-ide.org/current/index.html"

```

Comment:
```suggestion
    docs_link = __docs_url__
```

I think you forgot to make this change.

Reply:
```suggestion
```

I suggest simply removing it and using `__docs_url__` directly, it just adds extra complexity to redefine it here since its only used one place in this file.